package.json

```diff
@@ -23,7 +23,7 @@
     "build-dist-min": "webpack --bail --config node_modules/builder-victory-component/config/webpack/webpack.config.js --colors",
     "build-dist-dev": "webpack --bail --config node_modules/builder-victory-component/config/webpack/webpack.config.dev.js --colors",
     "build-dist": "builder run clean-dist && builder run build-dist-min && builder run build-dist-dev",
-    "build-lib": "builder run clean-lib && babel src -d lib --copy-files",
+    "build-lib": "builder run clean-lib && ('node_modules/builder-victory-component/node_modules/.bin/babel' src -d lib --copy-files || babel src -d lib --copy-files)",
```

Comment:
Something's very wrong if `node_modules/builder-victory-component/node_modules/.bin/babel` is needed.